### PR TITLE
Get Started Page

### PR DESCRIPTION
**Pull request by SarahPappas**
- Added an assets folder for images
- Added a InstructionModal component
- Add Instruction Modals to get started page
- Fixed background Images on get started page
- Show canvas only for routes under the pages\Trace folder
- Update css
- Add link for font on index.html

### CODE REVIEW

File: src/pages/Main/Main.tsx [hunk 1]
```diff
@@ -3,19 +3,21 @@ import './Main.css';
...
+    return <div className='pageContainer'><div className='mainContainer'>
 
-    return <div className='mainContainer'>
         {/* Add route here to make component visible*/}
-        <React.Fragment>
-            {renderCanvas()}
-        </React.Fragment>
+        <Route
+            path="/trace/:slug"
+            render={({match}) => {
```
Comment (msdrey) [inline src/pages/Main/Main.tsx:15]: add semicolon at line 16

File: src/pages/Main/Main.tsx [hunk 2]
```diff
@@ -3,19 +3,21 @@ import './Main.css';
...
+                return <CanvasContainer></CanvasContainer>
+            }}
+        />
         <Route exact path="/" component={GetStarted} />
         <Route exact path="/AddPhoto" component={AddPhoto} />
-    </div>;
-};
\ No newline at end of file
+        <Route exact path="/Trace/Instructions" component={Instructions} />
+    </div></div>
```
Comment (msdrey) [inline src/pages/Main/Main.tsx]: add semicolon at line 22

File: src/pages/Trace/Instructions/Instructions.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const button: Button = {text: "Get Started", to: "/AddPhoto" };
+
+export const Instructions: React.FC = () => {
+    return (
+        <div>Instructions</div>
+    );
+}
```
Comment (msdrey) [inline src/pages/Trace/Instructions/Instructions.tsx]: add semicolon at line 9

File: src/pages/GetStarted/GetStarted.tsx [hunk 4]
```diff
@@ -1,10 +1,22 @@
 import React from 'react';
 import { NavButton } from 'components/NavButton/NavButton';
+import { InstructionModal } from 'components/InstructionModal/InstructionModal';
+import backgroundArrow from '../../assets/background-arrow.svg';
+import './GetStarted.css';
+
+const button: Button = {text: "GET STARTED", to: "/AddPhoto" };
+const intro: InstructModal = {text: ["Let’s Make a \npattern in 3 \nsteps"]};
+const steps: InstructModal = {text: ["1. Upload photo", "2. Trace pattern", "3. Print pdf"]};
```
Comment (msdrey) [inline src/pages/GetStarted/GetStarted.tsx:10]: line 11 is extra

File: src/pages/GetStarted/GetStarted.tsx [hunk 5]
```diff
@@ -1,10 +1,22 @@
...
 
-const button: Button = {text: "Get Started", to: "/AddPhoto" };
 
 export const GetStarted: React.FC = () => {
-    return (
-        <NavButton button={button}/>
-    );
+    return (<>
+        <div className={'backgroundImageTop'}></div>
+        <img className={'backgroundImageBottom'} src={backgroundArrow}></img>
```
Comment (msdrey) [inline src/pages/GetStarted/GetStarted.tsx]: maybe adding an alt prop to the img component?

File: src/pages/Trace/Instructions/Instructions.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const button: Button = {text: "Get Started", to: "/AddPhoto" };
```
Comment (msdrey) [inline src/pages/Trace/Instructions/Instructions.tsx]: this button seems unnecessary

**Review by msdrey — COMMENTED** [review]
I added a few comments on small tiny details, missing semicolons or extra stuff. Other than that it looks good. That css looks like magic to me. Very impressed. I also really like how the 3 steps instructions are displayed in different divs using a map, very cool.

File: src/pages/Main/Main.tsx (hunk 1 above)
Comment (SarahPappas) [reply src/pages/Main/Main.tsx:15]: I forgot to run the linter with fix!

File: src/pages/GetStarted/GetStarted.tsx (hunk 5 above)
Comment (SarahPappas) [reply src/pages/GetStarted/GetStarted.tsx]: I saw that recommendation from the linter. The image is just the grey arrow in the background so its not very important, but I will add some alt text anyway.

File: src/pages/Trace/Instructions/Instructions.tsx (hunk 6 above)
Comment (SarahPappas) [reply src/pages/Trace/Instructions/Instructions.tsx]: The button will be used when we add the done button to this page. I added a todo at first, but I think its better to remove it for now.

**Pull request merged**